Tests/GeneratorEngineTests/EngineTests.swift

```diff
@@ -149,4 +149,35 @@ final class EngineTests: XCTestCase {
 
     try await engine.shutDown()
   }
+
+  struct MyItem: Sendable, CacheKey {
+    let remoteURL: URL
+    var localPath: FilePath
+    let isPrebuilt: Bool
+    init(remoteURL: URL, localPath: FilePath, isPrebuilt: Bool) {
+        self.remoteURL = remoteURL
+        self.localPath = localPath
+        self.isPrebuilt = isPrebuilt
+    }
```

Comment:
IIUC this is equivalent to the synthesized default initializer and is not needed?
```suggestion
```

Reply:
Ah, that was just for debugging purpose